src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -37,12 +37,12 @@ function compute_PairedExplicitRK3_butcher_tableau(num_stages, tspan,
 
     # Special case of e = 3
     if num_stages == 3
-        a_unknown = [0.25]
+        a_unknown = [0.25f0]
```

Comment:
Now, you will return a `Float32` array by default here. This may lead to type instabilities.